tests/testBasisLists.py

```diff
@@ -182,42 +184,42 @@ def testCentralRegularization(self):
         try:
             self.policyDF.set("centralRegularizationStencil", 1)
             ipDiffim.makeRegularizationMatrix(self.policyDF)
-        except Exception, e:  # stencil of 1 not allowed
+        except Exception as e:  # stencil of 1 not allowed
             pass
         else:
             self.fail()
 
         self.policyDF.set("centralRegularizationStencil", 5)
         try:
```

Comment:
I would remove all the boiler plate. The ones that are meant to work should just be the command that is meant to work. I don't feel that a bunch of boilerplate to translate the exception on failure to an AssertionError is worth the cost in extra code confusion. If @parejkoj objects, he can make the case for an `with self.assertDoesNotRaise():` construct that would at least make it obvious what the point is. The try/except/else/fail construct is just confusing. Look how much clearer the `with self.assertRaises()` versions are for the converse case.